src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -56,10 +56,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       // Remove user if in database.
       Key userKey = KeyFactory.createKey("User", email);
       try {
-        datastore.get(userKey);
+        Entity task = datastore.get(userKey);
         datastore.delete(userKey);
+        LOGGER.info("User was unsubscribed and removed from Datastore.");
       } catch (EntityNotFoundException e) {
-        LOGGER.log(Level.SEVERE, "User wants to unsubscribe but was never subscribed in the first place. ", e);
+        LOGGER.warn("User was unable to unsubscribe but was never subscribed.");
```

Comment:
does this mean "User was unable to unsubscribe because they were never subscribed"?

Reply:
Yes! An example would be if you, a new user, accidentally/wanted to unsubscribe but since you were never in the system, it will not let you unsubscribe.